tests/e2e/s3/02-init.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+unitTest: false
+commands:
+  - script: |
+      set -eux
+      export AWS_ACCESS_KEY_ID=minio
+      export AWS_SECRET_ACCESS_KEY=minio123
+      kubectl set env daemonset autoscaler-agent -n kube-system AWS_ACCESS_KEY_ID=minio
+      kubectl set env daemonset autoscaler-agent -n kube-system AWS_SECRET_ACCESS_KEY=minio123
+      
+      ../.venv/bin/python3 utils.py step2_init
+
+      # The reason we put the assert here as well is that a separate
+      # TestAssert step has different semantics:
+      #  1. It has a different home directory (root of the repo)
+      #  2. If the assert fails, the the assert step keeps retrying
+      #     until it succeeds or the test times out
```

Comment:
Maybe I'm missing something, but _presumably_ we'll be waiting for billing data to come in at some point, right? Wouldn't we want to retry that until the data is available?

Reply:
Hm, I see... fair enough. I guess any changes here can be left for a follow-up where we check correctness of the values we receive.